docs/git/setup.md

```diff
@@ -49,7 +49,7 @@ text-mode interface to Git.
 
 And, if none of the above are to your liking, try [one of these][gitbook-guis].
 
-[git-bash-admin]: ../development/setup-vagrant.md#running-git-bash-as-an-administrator
+[git-bash-admin]: ../development/setup-advanced.md#running-git-bash-as-an-administrator
```

Comment:
> If you are using Windows 10, make sure you [are running Git BASH as an administrator](http://localhost:9991/docs/development/setup-advanced.html#running-git-bash-as-an-administrator) at all times.

Good catch! Let's change this to "If you are using Windows 10 without WSL 2, ...."

<img width="659" alt="image" src="https://user-images.githubusercontent.com/2343554/188297177-6f2c7aae-ca0b-4b0e-bd94-972eceb0fa81.png">


> Also, since all contributors might end up here, do we want a link/phrase that sends them back to the development environment set up?

Yes, good idea.

<img width="732" alt="image" src="https://user-images.githubusercontent.com/2343554/188297185-a574ba99-1a76-4b6c-9daf-ee5ba6f0db68.png">